vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericWaitSteps.java

```diff
@@ -132,6 +140,66 @@ public String toString()
         });
     }
 
+    /**
+     * Waits for an <b><i>element</i></b> with the specified <b>locator</b> to stop moving
+     * @param locator locator to locate element
+     */
+    @When("I wait until element located by `$locator` stops moving")
+    public void waitForElementStopsMoving(Locator locator)
+    {
+        uiContext.getOptionalSearchContext().ifPresent(context ->
+            baseValidations.assertElementExists("The element to validate stop of its movement",
+                    context, locator).ifPresent(webElement -> {
+                        locator.getSearchParameters().setWaitForElement(false);
+                        IExpectedSearchContextCondition<Boolean> condition = new IExpectedSearchContextCondition<>()
+                        {
+                            private Point lastPosition;
+                            private boolean retryOnStaleAttempt;
+                            private WebElement movingWebElement = webElement;
+
+                            @Override
+                            public Boolean apply(SearchContext context)
+                            {
+                                try
+                                {
+                                    Point currentPosition = movingWebElement.getLocation();
+                                    boolean elementStoppedMoving = currentPosition.equals(lastPosition);
+                                    if (!elementStoppedMoving)
+                                    {
+                                        LOGGER.info("The element is moved at '{}'", currentPosition);
+                                    }
+                                    lastPosition = currentPosition;
+                                    return elementStoppedMoving;
+                                }
+                                catch (StaleElementReferenceException e)
+                                {
+                                    if (!retryOnStaleAttempt)
+                                    {
+                                        List<WebElement> movingWebElements = searchActions
+                                                .findElements(context, locator);
+                                        if (movingWebElements.size() == 1)
+                                        {
+                                            movingWebElement = movingWebElements.get(0);
+                                            retryOnStaleAttempt = true;
+                                            return false;
+                                        }
+                                    }
+                                    throw e;
+                                }
+                            }
+
+                            @Override
+                            public String toString()
+                            {
+                                return String.format("element located by \"%s\" stops moving",
```

Comment:
stopped moving?

Reply:
<img width="698" alt="image" src="https://github.com/vividus-framework/vividus/assets/18533593/ca7464cb-e7e7-4669-9db1-6e37ac35ff93">

Reply:
in this comment the comment from `@uarlouski` is valid

Reply:
updated